### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by sbabic**
Updated to 0.107.6
Support for RPI3 with SWUpdate and double-copy
Integrations added for 0.107.6

### CODE REVIEW

File: kas-rpi-master.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    url : "https://github.com/sbabic/meta-swupdate-boards.git"
+    refspec : master
+
+local_conf_header:
+  reduce_diskspace: |
+    INHERIT += "rm_work_and_downloads"
+    PATCHRESOLVE = "noop"
+    CONF_VERSION = "1"
+    DL_DIR ?= "/work/yocto_downloads"
+    SSTATE_DIR ?= "/work/sstate-cache-master"
```
Comment (bachp) [inline kas-rpi-master.yml]: This doesn't work in CI.

Is there a better way to set these?

File: recipes-homeassistant/images/core-image-homeassistant.bb [hunk 2]
```diff
@@ -2,6 +2,15 @@ SUMMARY = "A small image just capable of running Home Assistant"
...
 IMAGE_INSTALL = "packagegroup-core-boot ${CORE_IMAGE_EXTRA_INSTALL} python3-homeassistant python3-appdaemon"
 
+IMAGE_INSTALL += "\ 
+	libubootenv-bin \
+	mosquitto \
+	swupdate \
+	swupdate-progress \
+	swupdate-www \
+	swupdate-tools \
+	"
```
Comment (bachp) [inline recipes-homeassistant/images/core-image-homeassistant.bb]: This means that meta-swupdate also needs to be added to `kas-master.yml` and a kas file needs to be created for qemu.

File: kas-rpi-master.yml (hunk 1 above)
Comment (sbabic) [reply kas-rpi-master.yml]: We do not need DL_DIR and SSTATE_DIR - they can be removed form kas file, I'll do it.
Agree to remove warrior, then, and we just test master.

File: recipes-homeassistant/images/core-image-homeassistant.bb (hunk 2 above)
Comment (sbabic) [reply recipes-homeassistant/images/core-image-homeassistant.bb]: I propose to add them just for RPI3

Comment (bachp) [reply recipes-homeassistant/images/core-image-homeassistant.bb]: Also fine with me.

**Pull request closed without merging**